doc/visual-programming/source/widgets/data/featureconstructor.rst

```diff
@@ -12,36 +12,37 @@ Signals
 
 -  **Data**
 
-A data set.
+A data set
 
 **Outputs**:
 
 -  **Data**
 
-Modified data set.
+A modified data set
 
 Description
 -----------
 
-**Feature Constructor** allows you to manually add features (columns)
-into your data set. The new feature can be a computation of the existing
+The **Feature Constructor** allows you to manually add features (columns)
+into your data set. The new feature can be a computation of an existing
 one or a combination of several (addition, subtraction, etc.). You can
 choose what type of feature it will be (discrete, continuous or string)
 and what its parameters are (name, value, expression). For continuous
-variables you must define decimal precision and construct an expression
-in a Python mode.
+variables you only have to construct an expression in Python. 
 
 .. figure:: images/feature-constructor1-stamped.png
 
-1. List of constructed attributes
-2. Add or remove features
-3. New feature name
-4. Decimal precision
-5. Expression in Python
-6. Press commit to communicate changes
+   1. List of variable definitions 
+   2. Add or remove variables.   
+   3. New feature name
```

Comment:
Again on our own inconsistency: we should probably stick to either feature or variable.... No?